k8s/maintenance/phpmyadmin.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: phpmyadmin
```

Comment:
you need to put this in the same namespace as the rest of the things (e.g. so it can access the secrets) and not conflict with the phpmyadmin instances running for other Wordpress setups. Also so it can use the service name of `mariadb` without conflict.

Reply:
Good eye--will get that fixed.